src/Integration/Binding/BindingWorkflow.cs

```diff
@@ -62,6 +72,11 @@ public BindingWorkflow(IHost host, ProjectInformation project)
 
         #region Workflow state
 
+        public List<Project> BindingProjects
```

Comment:
Can you bind the same project more than once? Do we need to bind in order? if not using a hash set might better represent that.